app/src/main/java/com/babylon/wallet/android/presentation/settings/appsettings/gateways/GatewaysViewModel.kt

```diff
@@ -105,14 +103,14 @@ class GatewaysViewModel @Inject constructor(
             _state.update { state -> state.copy(addingGateway = true) }
 
             val newGatewayInfo = networkInfoRepository.getNetworkInfo(newUrl)
-            newGatewayInfo.onValue { networkName ->
+            newGatewayInfo.onSuccess { networkName ->
```

Comment:
like Michael mentioned above this can be `newGatewayInfo.onSuccess{}.onFailure{}`